packages/sls-aws/src/eventbridge/index.ts

```diff
@@ -5,7 +5,7 @@ import { jsonEventBridgeEvent } from './json-eventbridge-event'
 import { jsonEventBridgeMessage } from './json-eventbridge-message'
 
 export type EventBridgeEventHandler = Handler<EventBridgeEvent<string, never>, Context, Promise<void>>
-export type EventBridgeMessageHandler = Handler<EventBridgeEvent<string, string>, Context, Promise<void>>
+export type EventBridgeMessageHandler = Handler<EventBridgeEvent<string, any>, Context, Promise<void>>
```

Comment:
ideally, we would be able to define the payload of `EventBridgeEvent` and have a specific `T` around it 

that would have to be a different type than `Payload` as they both can be different